Code/GraphMol/new_canon.cpp

```diff
@@ -567,6 +567,9 @@ void updateAtomNeighborNumSwaps(
 
 void rankMolAtoms(const ROMol &mol, std::vector<unsigned int> &res,
                   bool breakTies, bool includeChirality, bool includeIsotopes) {
+  if (!mol.getNumAtoms())
```

Comment:
Should probably move the `res.resize(mol.getNumAtoms());` that's a few lines lower to be the first line in the function. That way it will be returned with the appropriate size, even if there are zero atoms.